FRAuth/FRAuth/OAuth2/AccessToken.swift

```diff
@@ -168,6 +168,23 @@ import Foundation
         aCoder.encode(self.sessionToken, forKey: "session_token")
     }
     
+    private enum CodingKeys: String, CodingKey {
+            case value, expiresIn, scope, tokenType, refreshToken, idToken, authenticatedTimestamp, sessionToken
```

Comment:
For the same reason below, the `value` may not be needed as it will be encoded with `Token` class? I maybe wrong on this, let me know if you think otherwise.

Reply:
Will have a look to verify

Reply:
updated